xbmc/utils/BitstreamConverter.cpp

```diff
@@ -279,19 +279,27 @@ static bool has_sei_recovery_point(const uint8_t *p, const uint8_t *end)
 #ifdef HAVE_LIBDOVI
 // The returned data must be freed with `dovi_data_free`
 // May be NULL if no conversion was done
-static const DoviData* convert_dovi_rpu_nal(uint8_t* buf, uint32_t nal_size, int mode)
+static const DoviData* process_dovi_rpu_nal(uint8_t* buf, uint32_t nal_size, int mode, int& dovi_el_type)
```

Comment:
I don't think we should mess with Kodi's code too much here.
It may change in the future and we'll have to rebase it manually. I would think if there's a way to wrap this somehow in a dedicated method or separate block of code.